alacritty_terminal/src/term/mod.rs

```diff
@@ -1586,7 +1586,13 @@ impl<T: EventListener> Handler for Term<T> {
 
     #[inline]
     fn move_forward_tabs(&mut self, count: u16) {
-        trace!("[unimplemented] Moving forward {} tabs", count);
+        trace!("Moving forward {} tabs", count);
+
+        self.put_tab(count);
+        let cell = self.grid.cursor_cell();
+        if cell.c == ' ' {
+            self.damage_cursor();
+        }
```

Comment:
Could you please just apply the patch I've send before with `git apply -3`?

Reply:
k it makes sense now 😅